documentation/ide-vscode-setup.html

```diff
@@ -0,0 +1,189 @@
+            This will quickly import the Openfire source code into  VS Code .
+        </p>
+
+        <figure>
+            <a href="images/VSCode-2.png"><img src="images/VSCode-2.png" alt="Use the 'Open' functionality of VS Code."></a>
+            <a href="images/VSCode-3.png"><img src="images/VSCode-3.png" alt="Select the folder that will open in our VS Code."></a>
+            <a href="images/VSCode-4.png"><img src="images/VSCode-4.png" alt="Overview of project in Visual Studio Code"></a>
+	    <figcaption>Opening Openfire Source folder on VS Code.</figcaption>            
+        </figure>
+
+        <p>
+            VS Code will automatically recognize and process the Maven structure of Openfire (pom.xml file) and it will automatically download all dependencies.
+        </p>
+        <h2>Build Openfire from VS Code</h2>
+        <p>
+            To build the code straight from your IDE. The following instructions will explain how
+            you can build Openfire from the code that is in the VS Code workspace.
+        </p>
+        <p>
+            Select the 'New Terminal' option -- <b>Terminal > New Terminal</b> to open a terminal and run the command bellow to  build the complete project including plugins: 
+            <ul>
+                <span class="code"> mvn verify <br></span>
+            </ul> 
+            This will quickly builds Openfire (compile,runs all unit tests) and creates a distributable package 
+            in the <tt>distribution</tt> module in <tt>distribution/target/</tt>  folder. 
+        </p>
+    
+        <figure>
+            <a href="images/VSCode-5.png"><img src="images/VSCode-5.png" alt=""></a>
+            <a href="images/VSCode-6.png"><img src="images/VSCode-6.png" alt=""></a>
+            <a href="images/VSCode-7.png"><img src="images/VSCode-7.png" alt=""></a>
+            <figcaption>Build Openfire from the source code.</figcaption>
+        </figure>
+
+	    
+        <h2>Run Openfire from VS Code</h2>
+        <p>
+            When developing, it is often also useful to run the code straight from your IDE.
+            You can launch Openfire using the scripts in <tt>distribution/target/distribution-base/bin</tt> by using the terminal vscode.
+        </p>
+        <p>
+            <ul>
+                <span class="code"> sh distribution/target/distribution-base/bin/Openfire.sh <br></span>
+            </ul> 
+        </p>
+
+        <figure>
+            <a href="images/VSCode-8.png"><img src="images/VSCode-8.png" alt="Launch Openfire server  using shell script."></a>
+            <a href="images/VSCode-9.png"><img src="images/VSCode-9.png" alt="The Openfire startup screen after launching the server."></a>
+            <figcaption>Run Openfire straight from the source code.</figcaption>
+        </figure>
+
+        <p>
+            After running Openfire, you can go to your URL server using your browser to install and configure Openfire.
```

Comment:
Comma not needed after Openfire and
"you can go to your server using the URL shown in console (or http://localhost:9090) and configure it."

Reply:
Comma not needed after "After running Openfire"